setup/kubernetes/aws.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Set Up an Amazon Elastic Kubernetes Cluster
+description: Learn how to set up an Amazon EKS cluster for your Coder deployment.
+---
+
+This deployment guide shows you how to set up an Amazon Elastic Kubernetes
+Engine cluster on which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following utilities
+installed on your machine:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [AWS Command Line
+  Interface](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+  (you'll also need to
+  [configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+  the the command line interface to interact with your AWS account; consider
+  AWS' [CLI configuration
+  quickstart](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+  to fast-track this process))
+- [eksctl command line
+  utility](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+
+## Preliminaries
+
+1. Go to AWS' [EC2 Console](https://us-east-2.console.aws.amazon.com/ec2/); this
```

Comment:
```suggestion
1. Go to AWS' [EC2 Console](https://console.aws.amazon.com/ec2/); this
```

Updating URL as we say it should take you to your AWS region you're working in, but it was set to a specific region in the URL. The above will redirect to the current region (aka last region you were using when logged in)